bin/envconfig.csh

```diff
@@ -137,6 +137,12 @@ conda activate "$LSST_CONDA_ENV_NAME"
 
 source "$LSSTSW/eups/current/bin/setups.csh"
 
+# definition EUPS_PATH depending on the environment:
+# including the last 7 characters of the environment name
+set env_ref = `echo $LSST_CONDA_ENV_NAME | awk '{print substr($0, length($0) - 6)}'`
+set EUPS_PATH="$LSSTSW/stack$env_ref"
```

Comment:
As above, replace this with `set EUPS_PATH="$LSSTSW/stack-$LSST_SPLENV_REF"`?

Reply:
Better using subfolders